cmd/minikube/cmd/start.go

```diff
@@ -156,6 +169,36 @@ func runStart(cmd *cobra.Command, args []string) {
 	clusterUp(&config)
 }
 
+// Set docker and oc cluster proxy in place
+func setProxy() {
+	if viper.IsSet("http-proxy") {
```

Comment:
That the proxy matches the expected format - http://fedora:fedora123@10.70.49.109:3128
In fact, the help string should mention mention this very format, so that the user knows how to set the proxy.